### DISCUSSION TITLE
mspider/user-guide/sidecar-management/workload-sidecar

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3701] [Announcements]
# mspider/user-guide/sidecar-management/workload-sidecar

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/sidecar-management/workload-sidecar.html

**Comment by nicole-lihui** [comment]
建议对工作负载类型进行描述和解释
